picasso/src/main/java/com/squareup/picasso/PicassoProvider.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2017 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.picasso;
+
+import android.content.ContentProvider;
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.net.Uri;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+public class PicassoProvider extends ContentProvider {
+  @Override public boolean onCreate() {
+    Picasso.setSingletonInstance(new Picasso.Builder(getContext()).build());
```

Comment:
if you want to override the singleton instance, then, you need to remove the ContentProvider from the manifest, right?
or is there another way?